setup.py

```diff
@@ -0,0 +1,34 @@
+import distutils.core
+from distutils.command.install import install
+import subprocess
+
+
+class PyenvInstall(install):
+    def run(self):
+        print(subprocess.check_output(['bash', 'bin/pyenv-installer']))
+
+with open('README.rst') as file:
+    long_description = file.read()
+
+distutils.core.setup(
+    version='20150113',
```

Comment:
... at the moment that version number is useless though as pyenv does not turn up in pip freeze as it is atm :( That's another thing I want to find out soonish to make that integrate into the standard Python toolchain better